README.md

```diff
@@ -32,7 +32,12 @@
 
```

Comment:
![image](https://github.com/Grandi0z/school_library/assets/93039359/d9053d99-f186-426b-80e5-4ec5d3df1b31)

- [ ] It would be nice if you removed this title because you already have a title after the Table of Contents section so it would be better if you remove it to make your readme more professional 👍